src/app/containers/Image/index.test.jsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import ImageContainer from './index';
-import { shouldShallowMatchSnapshot, isNull } from '../../../testHelpers';
+import {
+  shouldShallowMatchSnapshot,
+  isNull,
+  suppressPropWarnings,
+} from '../../../testHelpers';
 import { blockContainingText, blockArrayModel } from '../../models/blocks';
 
 describe('Image', () => {
   describe('with no data', () => {
+    suppressPropWarnings(['blocks is not an array']);
```

Comment:
Does `suppressPropWarnings` affect every test in this file/suite? We need it to reset itself ASAP after its used really. 
```
describe('with no data one', () => {
  suppressPropWarnings(['blocks is not an array']);
  isNull('should return null', <ImageContainer />);
});

describe('with no data two', () => {
  isNull('should return null', <ImageContainer />);
});
```
should throw an error for the second one.

Maybe we need a `resetSuppressedPropWarnings` function that cleans window.expectedWarnings that you can run in a `afterEach()`? Can this be done automatically?

Reply:
No. `supressPropWarnings` is automatically removed after suppression in the `src/testHelpers/setupTests.js` file.
https://github.com/bbc/simorgh/blob/bc2414bad622858828373a83473c753ad6fa6bbe/src/testHelpers/setupTests.js
```
if (expectedWarnings && typeof expectedWarnings === 'object') {
    const warningsRegex = new RegExp(
      [REACT_FAILED_PROP_TYPE, ...Object.values(expectedWarnings)].join('.*'),
    );
    if (warningsRegex.test(message)) {
      Object.defineProperty(window, 'expectedWarnings', {
        value: undefined,
        writable: true,
      });
      return;
    }
  }
```

Reply:
Interesting, I tried
```
describe('with no data one', () => {
  suppressPropWarnings(['blocks is not an array']);
  isNull('should return null', <ImageContainer />);
});

describe('with no data two', () => {
  isNull('should return null', <ImageContainer />);
});
```
Which didnt throw a prop warning at all
and
```
describe('with no data one', () => {
  isNull('should return null', <ImageContainer />);
});

describe('with no data two', () => {
  suppressPropWarnings(['blocks is not an array']);
  isNull('should return null', <ImageContainer />);
});
```
which did

Reply:
@dr3 I think it might be because react doesn't trigger warnings on rerender https://github.com/facebook/react/issues/4302

Reply:
@sadickisaac That's true. @dr3 This particular example failed because the console message was exactly the same. Which we shouldn't be testing for anyway, so I think we are good to go

Reply:
@pharingee But doesnt that mean if 2 tests emit 1 required prop, for different reasons, they will ignore the second? I really think we should try and make this as isolated as possible

Reply:
The current implementation is as isolated as possible. If two tests emit the same warning in the same context, only one of the warnings will be printed. This behaviour is from react and is independent of the `suppressPropWarnings` function. As such, it is beyond the scope of this PR.